packet/src/lib.rs

```diff
@@ -300,6 +302,20 @@ impl Meta {
     pub fn is_from_staked_node(&self) -> bool {
         self.flags.contains(PacketFlags::FROM_STAKED_NODE)
     }
+
+    #[inline]
+    pub fn remote_pubkey(&self) -> Option<Pubkey> {
+        if self.remote_pubkey == Pubkey::default() {
+            None
+        } else {
+            Some(self.remote_pubkey)
+        }
+    }
+
+    #[inline]
+    pub fn set_remote_pubkey(&mut self, pubkey: Pubkey) {
+        self.remote_pubkey = pubkey;
```

Comment:
nit: Up to you if this is the API you want to use, but it might be good flagging that passing `Pubkey::default()` results in nulling the remote pubkey in a doc comment

Reply:
added comment